.dockerignore

```diff
@@ -1,10 +1,29 @@
+**.DS_Store
+.env
+.devcontainer
+.dockerignore
+.editorconfig
 .git
-node_modules
-vendor
+.github
+**.gitignore
+.php-cs-fixer.dist.php
+.prettierrc.json
+.vscode
+Dockerfile
+bounties.md
+compose.yml
+contributing.md
+contributor_license_agreement.md
 database/database.sqlite
+docker/README.md
+node_modules
+phpstan.neon
+phpunit.xml
+readme.md
```

Comment:
The main readme can be in the docker container, that is not an issue

Reply:
But is it useful to provide it? Excluding it does provide the benefit that the container won't be rebuilt on any changes to readme. Shouldn't really matter either way though.

Reply:
it is a dockerignore? the workflow will still trigger on every push to the main branch?

Reply:
Oh no, I didn't mean it would entirely skip it. I meant that it would just cache every layer and the build would be done almost instantly. But again, it doesn't make a huge difference anyways, as it's not like the readme is updated super often. Can also just be excluded from the building steps, then added in the end to minimize rebuilding.

Another option is the Github action could also just ignore changes to readme, so that the image doesn't need rebuilding if the readme gets updated or something.

Reply:
It is fine, I do get why you excluded it. You can leave it out